File: src/main/java/org/jdc/template/model/webservice/websearch/dto/DtoPage.java
package org.jdc.template.model.webservice.websearch.dto;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class DtoPage {
    @JsonField
    private String start;
    @JsonField
    private int label;

    public String getStart() {
        return start;
    }

    public void setStart(String start) {
        this.start = start;
    }

    public int getLabel() {
        return label;
    }

    public void setLabel(int label) {
        this.label = label;
    }
}

File: src/main/kotlin/org/jdc/template/model/webservice/websearch/dto/DtoPage.kt
package org.jdc.template.model.webservice.websearch.dto

import com.bluelinelabs.logansquare.annotation.JsonField
import com.bluelinelabs.logansquare.annotation.JsonObject

@JsonObject
class DtoPage {
    @JsonField
    var start: String? = null
    @JsonField
    var label: Int = 0
}
